src/main/resources/api/api.yaml

```diff
@@ -250,6 +404,79 @@ definitions:
   #
   # Questions
   #
+  V1Question:
+    type: object
+    required:
+      - text
+      - type
+      - topic
+    properties:
+      id:
+        type: integer
+        format: int64
+      text:
+        type: string
+      type:
+        type: string
```

Comment:
integer

Reply:
So we will do it integer in API? And where we will see string question type?

Reply:
If you want to implement mapping from int to string (as we decide at pair), than it must be string, but example should not be '1', '2'
Otherwise it must be int, and example should be 1, 2, 3 etc...